pkg/networkplugin-syncer/handlers/ovn/nbctl/nbctl.go

```diff
@@ -109,3 +116,34 @@ func parseLrPolicyGetOutput(output, rerouteIp string) *util.StringSet {
 
 	return subnets
 }
+
+// expand strings in the form of
+// ssl:ovnkube-db.openshift-ovn-kubernetes.svc.cluster.local:9641 to
+// ssl:<ip1>:9641,ssl:<ip2>:9641,ssl:<ip3>:9642
```

Comment:
This should be 9641 at the end, shouldn’t it?

Reply:
ooops yes.

Reply:
Done